crates/miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -333,17 +333,17 @@ end
 
 #! Sets the account ID.
 #!
-#! Inputs:  [acct_id_prefix, acct_id_suffix]
-#! Outputs: []
+#! Inputs:  [acct_id_prefix, acct_id_suffix, block_num]
+#! Outputs: [block_num]
 #!
 #! Where:
 #! - acct_id_{prefix,suffix} are the prefix and suffix felts of the account ID.
 export.set_global_acct_id
     push.0.0
-    # => [0, 0, acct_id_prefix, acct_id_suffix]
+    # => [0, 0, acct_id_prefix, acct_id_suffix, block_num]
     mem_storew.ACCT_ID_PTR
     dropw
-    # => []
+    # => [block_num]
 end
```

Comment:
I don't think we need to make these updates here because `set_global_acct_id` doesn't really care about what else is on the stack besides the `acct_id_prefix, acct_id_suffix`.